Kernel/makeall.sh

```diff
@@ -7,7 +7,7 @@ if [ -z "$MAKEJOBS" ]; then
     MAKEJOBS=$(nproc)
 fi
 
-make_cmd="make -j $MAKEJOBS"
+make_cmd="bear make -j $MAKEJOBS"
```

Comment:
This seems very wrong

Reply:
Ah, that wasn't supposed to make it in...